client/packages/core/tsconfig.json

```diff
@@ -9,6 +9,7 @@
     "outDir": "dist",
     "rootDir": "src",
     "sourceMap": true,
+    "inlineSources": true,
```

Comment:
Will inlineSources increase the size of the bundle we publish to npm?

Reply:
Afaik it will not. Here's what I did: 

1. Asked chatgpt: https://chatgpt.com/share/66ec6a75-9ca0-8003-ae2a-916d472ba29a 
2. Looked at the `dist` folder. It looks like all the js files just have a single comment at the bottom, pointing to a .map file. Afaik I don't think those .map files will ship on a production bundle